DevSkim-DotNet/Microsoft.DevSkim.CLI/Options/SuppressionCommandOptions.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using CommandLine;
+
+namespace Microsoft.DevSkim.CLI.Options;
+
+[Verb("suppressions", HelpText = "Apply suppressions from a Sarif")]
+public class SuppressionCommandOptions
+{
+    [Option('I',Required = true, HelpText = "Path to source code")]
+    public string Path { get; set; } = String.Empty;
+    [Option('O',Required = true, HelpText = "Filename for input sarif with proposed fixes.")]
```

Comment:
Nit: I think you copied this description from the Apply Fixes command, but here the fixes are not necessary for the suppresions.
```suggestion
    [Option('O',Required = true, HelpText = "Filename for sarif with DevSkim scan results.")]
```